lib/api_callable.js

```diff
@@ -121,12 +121,14 @@ Canceller.prototype.call = function(aFunc, argument) {
 /**
  * PromiseCanceller is Canceller, but it holds a promise when
  * the API call finishes.
+ * @param {Module} PromiseModule - A module that implements the ES6 specification of
```

Comment:
I'm feeling like this is not a module in JS vocabulary. Rather it's a constructor of promise.  The type will be a `Function` (because no "constructor" type in JSDoc), and the symbol name might be `PromiseCtor` or something.